satellite/accounting/live/plainmemory.go

```diff
@@ -56,48 +23,57 @@ type plainMemoryLiveAccounting struct {
 	log *zap.Logger
 
 	spaceMapLock sync.RWMutex
-	spaceDeltas  map[uuid.UUID]spaceUsedAccounting
-}
-
-type spaceUsedAccounting struct {
-	inlineSpace int64
-	remoteSpace int64
+	spaceDeltas  map[uuid.UUID]int64
 }
 
 func newPlainMemoryLiveAccounting(log *zap.Logger) (*plainMemoryLiveAccounting, error) {
 	pmac := &plainMemoryLiveAccounting{log: log}
-	pmac.ResetTotals()
+	pmac.spaceMapLock.Lock()
+	pmac.spaceDeltas = make(map[uuid.UUID]int64, 0)
+	pmac.spaceMapLock.Unlock()
```

Comment:
there's no need for the lock here since nothing else can be holding a reference to it.